tests/core/core.spec.ts

```diff
@@ -901,7 +901,7 @@ suite("@dataform/core", () => {
       expect(
         cGraph.graphErrors.compilationErrors.filter(item =>
           item.message.match(
-            /Ambiguous Action name: {\"name\":\"a\"}. Did you mean one of: foo.a, bar.a./
+            /Ambiguous Action name: {\"name\":\"a\",\"includeDependentAssertions\":false}. Did you mean one of: foo.a, bar.a./
```

Comment:
Note to self: this is kind of gross, we should fix it by not using JSON stringify like this https://github.com/dataform-co/dataform/blob/754d582f4f588b3c812c40a38ec839607ec3d4f0/core/utils.ts#L129C42-L129C53